arena.js

```diff
@@ -0,0 +1,47 @@
+import Gladiator from './gladiator.js';
+
+
+export default class Arena extends Gladiator {
+    constructor(name, gladiators = []) {
+        super();
+        this.name = name.toUpperCase();
+        this.gladiators = gladiators;
+    }
+
+
+    addGladiator(gladiators) {
+        if (this.gladiators.length < 2)
+            this.gladiators.push(gladiators);
+    }
+
+
+    fight() {
+
+        if (this.gladiators.length >= 1) {
+            if ((this.gladiators[0].weapon === 'Trident' && this.gladiators[1].weapon === 'Spear') ||
+                (this.gladiators[0].weapon === 'Spear' && this.gladiators[1].weapon === 'Club') ||
+                (this.gladiators[0].weapon === 'Club' && this.gladiators[1].weapon === 'Trident')
+            ) {
+
+            } //inner if
+            else if ((this.gladiators[1].weapon === 'Trident' && this.gladiators[0].weapon === 'Spear') ||
+                (this.gladiators[1].weapon === 'Spear' && this.gladiators[0].weapon === 'Club') ||
+                (this.gladiators[1].weapon === 'Club' && this.gladiators[0].weapon === 'Trident')
+            ) {
+
+            } //inner if
+            else if ((this.gladiators[0].weapon === 'Trident' && this.gladiators[1].weapon === 'Trident') ||
+                (this.gladiators[0].weapon === 'Spear' && this.gladiators[1].weapon === 'Spear') ||
+                (this.gladiators[0].weapon === 'Club' && this.gladiators[1].weapon === 'Club')
+            ) {
+                this.gladiators[0].pop();
```

Comment:
.pop() does not work here. pop is a method that only works on an array but not on an element.

one way to remove an element is to use a method called .splice()
here is a link on how to use it
[splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)